thanos/templates/query-deployment.yaml

```
@@ -53,6 +56,11 @@ spec:
           containerPort: {{ .Values.cluster.port }}
         resources:
           {{ toYaml .Values.query.resources | nindent 10 }}
+        {{- if .Values.query.certSecretName }}
+        volumeMounts:
+        - mountPath: /etc/certs
+          name: {{ .Values.query.certSecretName }}
+          readOnly: true
```

Fix missing {{ end }}

```suggestion
          readOnly: true
        {{- end }}
```